Content: Unfettered campaign Act 1 (#6416)
endless-sky/endless-sky: data/hai/mercenaries.txt

	description "	This version was built from stronger materials, allowing a stronger hull, at the cost of heat dissipation. The owners also made modifications in order to make this ship more agile and flexible, with increased fuel and engine capacity. It is a terror on the battlefield, using its speed and high damage of the Tripulse Shredders to its advantage."

fleet "Marauders"
	government "Mercenaries"
	names "pirate"
	personality
		disables plunders harvests
	variant 2
		"Marauder Firebird (Engines)"
	variant 3
		"Marauder Firebird"
	variant 3
		"Marauder Arrow (Weapons)"
		"Marauder Bounder"
	variant 3
		"Marauder Falcon"
	variant 3
		"Marauder Raven (Weapons)"
	variant
		"Marauder Raven"
		"Marauder Quicksilver (Weapons)" 2
	variant 3
		"Marauder Splinter (Weapons)"
	variant 2
		"Marauder Manta" 2
	variant
		"Marauder Leviathan (Engines)"
	variant
		"Marauder Raven" 2
	variant
		"Sea Scorpion (Ion)"
	variant
		"Sea Scorpion (Pulse)"
	variant
		"Lightning Bug" 2

> phrase "mercenaries defeated" 

Reviewer: ```suggestion
phrase "mercenaries defeated"
```

			choice
				`(Sell the bars.)`
				`(Keep the bars, and drop that traitor on the planet.)`
					goto bliss2
				`(Keep the crew member for further interrogation.)`
					goto escape
			action
				outfit "Gold Bars" -5
				payment 5000000
				set "mercenaries: sold bars"
			`The soon-to-be gone traitor nods. "It is the right choice you are making, Captain <last>. You will never be bothered by any of this again, that I promise. I will take these 5 piles, and any more will be sold little by little as you go on most worlds. If you sell them to any normal outfitters, we will easily get our hands on it, so it is no problem to us, but you would only be paid the price of brute gold for it."`
				decline
			label bliss2
			`You leave the traitor here on this world after awakening them, and they seem thankful but deeply troubled by what happened. You wonder what you should do with the gold bars that seem to cause so much trouble. They should probably be stashed away safely, or maybe you should get rid of them.`
				decline
			label escape

> 			`You tighten your hold on the traitor, but as you do something hits the back of your head, and you lose consciousness. When you wake, there is a small note on the spot where some gold bars were, written by yet another member of  your crew. "<first>, I am very sorry to break your trust like they did, but believe me, this is for your own good. The ex-crew member would have been killed if they had returned without the gold bars, so we took a few. You should quit this while you still can."`

Reviewer: ```suggestion
			`You tighten your hold on the traitor, but as you do something hits the back of your head, and you lose consciousness. When you wake, there is a small note on the spot where some gold bars were, written by yet another member of your crew. "<first>, I am very sorry to break your trust like they did, but believe me, this is for your own good. The ex-crew member would have been killed if they had returned without the gold bars, so we took a few. You should quit this while you still can."`
```

		
		"Wanderer Ramscoop"
		"Fuel Processor"
		"Systems Core (Medium)"
		"Geode Reactor"
		"Hai Valley Batteries"
		"Value Detector"
		
		Hyperdrive
		"Ionic Blaster Prototype" 4
		"Ionic Turret Prototype" 2
		"Quantum Keystone"
		"Ionic Blade"
		"Pulse Rifle" 61
		"Dual Pulse Pistols" 5
	cargo
		outfits
			"Gold Bars" 50
	"steering engine" 62 10
		zoom 0.6
		right
		angle -115
	"steering engine" -62 10
		zoom 0.6
		left
		angle 115
	"reverse engine" 0 -96
		zoom 0.8
	engine 44.5 64
		zoom 0.8
	engine -44.5 64
		zoom 0.8
	engine 0 110
	gun -24 -100 "Ionic Blaster Prototype"
	gun 24 -100 "Ionic Blaster Prototype"
	gun -15 -97 "Ionic Blaster Prototype"
	gun 15 -97 "Ionic Blaster Prototype"
	turret -22.5 -57.5 "Chameleon Anti-Missile"
	turret 22.5 -57.5 "Chameleon Anti-Missile"
	turret -15 41 "Ionic Turret Prototype"
	turret 15 41 "Ionic Turret Prototype"
	bay Drone -67 -76
		under
	bay Drone 67 -76
		under
	bay Drone 0 70.5
		over
	leak leak 40 50
	leak flame 50 80
	leak "big leak" 60 50
	explode "huge explosion" 10
	explode "large explosion" 30
	explode "medium explosion" 50
	explode "small explosion" 40
	explode "tiny explosion" 40
	"final explode" "final explosion large"

> 	description "This ship was built near the end of the Korath raids to be fast enough to catch and delay the raiders of the time. Whilst ordinarily it could not defeat one alone, its energy weapons and drones let it hold its own until cavalry arrived, and sometimes even win the fight if lucky. It is not able to sustain prolonged combat, but it is now renowned for attacking vulnerable targets in human space and disappearing before anyone can react, as if always one move ahead."

Reviewer: ```suggestion
	description "This ship was built near the end of the Korath raids to be fast enough to catch and delay the raiders of the time. Whilst ordinarily it could not defeat one alone, its energy weapons and drones let it hold its own until the cavalry arrived, and sometimes even win the fight if they got lucky. It is not able to sustain prolonged combat, but it is now renowned for attacking vulnerable targets in human space and disappearing before anyone can react, as if always one move ahead."
```
This description needs more work.

Author: it's hot now

	description "	When the raids suddenly ended, this ship turned out to be less useful in the subsequent Unfettered conflict, and was stolen by Mercenaries who chose to remain neutral in the conflict."

ship "Tick"
	sprite "ship/hai flea"
	thumbnail "thumbnail/hai flea"
	attributes
		category "Drone"
		"cost" 186500
		"shields" 200
		"hull" 1000
		"mass" 24
		"drag" .55
		"heat dissipation" .7
		"outfit space" 63
		"weapon capacity" 14
		"engine capacity" 22
		"automaton" 1
		weapon
			"blast radius" 8
			"shield damage" 65
			"hull damage" 37
			"hit force" 85
	outfits
		"Korath Repeater"
		"Hai Chasm Batteries" 2
		"Small Biochemical Cell"
		`"Baellie" Atomic Engines`

	engine 0 27 1.4
	gun 0 -27
	turret 0 -15
	leak "leak" 60 50
	explode "tiny explosion" 17

> 	description "This drone was customized with the idea of using either a Repeater, to complement ion attacks, or a Bullfrog Anti-Missile, to keep its owner safe. The denser materials used allow for better protection, at the cost of a bit of mass and heat dissipation."

Reviewer: ```suggestion
	description "This drone was customized with the idea of using either a Repeater, to complement ion attacks, or a Bullfrog Anti-Missile, to keep its owner safe. The denser materials used allow for better protection, at the cost of a bit more mass and lower heat dissipation."
```

			"hull damage" 520
			"hit force" 1500
	outfits
		`"Bondir" Atomic Thruster`
		`"Bondir" Atomic Steering`
		"Boulder Reactor"
		"Hai Diamond Regenerator" 2
		"Hai Gorge Batteries"
		"Hai Williwaw Cooling" 3
		"Jump Drive"
		"Outfits Expansion" 2
		"Pulse Rifle" 35
		"Dual Pulse Pistols"
		"Tripulse Shredder" 4
		"Wanderer Ramscoop"
	cargo
		outfits
			"Gold Bars" 5
	engine -28.5 68.5
	engine 28.5 68.5
	"steering engine" 64.5 23.5
		zoom 0.6
		angle -90
		right
	"steering engine" -64.5 23.5
		zoom 0.6
		angle 90
		left
	gun -13 -77.5
	gun 13 -77.5
	gun -27 -56
	gun 27 -56
	turret -35.5 24.5
	turret 35.5 24.5
	leak "leak" 40 50
	leak "flame" 40 30
	explode "tiny explosion" 30
	explode "small explosion" 50
	explode "medium explosion" 35
	explode "large explosion" 18
	"final explode" "final explosion medium"
	description "The Sea Scorpion was designed to serve as a mobile platform to launch Trackers from. After decades of hard work, the Unfettered are finally able to mass produce them. The ever growing fleet of Sea Scorpions enables the Unfettered to employ new tactics against the other Hai which alleviate some of their numerical, technological, and logistical disadvantages."

> 	description "	This version was built from stronger materials, allowing a stronger hull, at the cost of heat dissipation. The owners also made modifications in order to make this ship more agile, and flexible, with increased fuel and engine capacity. It is a terror on the battlefield, using its speed and high damage of the Tripulse Shredders to its advantage."

Reviewer: ```suggestion
	description "	This version was built from stronger materials, allowing a stronger hull, at the cost of heat dissipation. The owners also made modifications in order to make this ship more agile and flexible, with increased fuel and engine capacity. It is a terror on the battlefield, using its speed and high damage of the Tripulse Shredders to its advantage."
```

		not "mercenaries: took out leader"
		random < 1
	npc disable
		government "Mercenaries"
		system
			distance 2 5
		fleet "Marauders" 5
		dialog phrase "mercenaries defeated"
	to fail
		or
			has "mercenaries: took out leader"
			not "outfit: Gold Bars"

# SPOILER: mercenaries sometimes save pirates from slavery, in exchange for a life debt.
# Some of your crew may be part of that, or maybe it is just any pirate from this world that tries to get them, attracted by the huge reward.
mission "Mercenaries: Betrayal For The Bars"
	landing
	invisible
	source
		government "Pirate"
	to offer
		random < 40
		has "Unfettered: Jump Drive Retrieval: done"
		has "flagship crew" > 4
		"outfit (flagship installed): Gold Bars" + "outfit (flagship cargo): Gold Bars" >= 5
	on offer
		conversation
			`As you land on the pirate world, you head for the hatch, preparing to enter the spaceport, but passing by the cargo bay you think you hear a sound. Odd... There shouldn't be anyone there right now, knowing your crew's habits, as most of them have already disembarked.`
			choice
				`(Its nothing. Probably just a rat.)`
				`(Go investigate.)`
					goto investigate
			action
				outfit "Gold Bars" -5

> 			``

Reviewer: What is this for? Can't you just defer in `choice`?

Author: I need to take the bars first, and `` is needed to have an endpoint

				defer
			label investigate
			`You pull out your sidearm and approach the bay as discreetly as you can. Worst case scenario is you find one of your own crew, in which case you better not overreact, or someone could get hurt.`
			choice
				`(Ask if someone is there.)`
					goto hello
				`(Never too careful.)`
			`You go into the cargo bay, on guard. As you enter, you think you see something move, on your left. There is not much time to think.`
			choice
				`(Call my crew to help.)`
				`(Run in that direction.)`
					goto run
			label help
			`You call your crew to help and try to barricade the ship to prevent escape, but they cannot find anyone. You wonder how they could get away. Just to be sure, you check around the cargo bay. It seems like nothing is missing, but a few bars of gold are out of place. Maybe it was just a malfunction with the ship's gravitational systems.`
				defer
			label run

> 			`As you run, you see a silouette running too. They are armed, and you think you recognize the uniform of a member of your crew. Could they be running after the intruder too? Is this just a big misunderstanding?`

Reviewer: ```suggestion
			`As you run, you see a silhouette of someone running. They are armed, and you think you recognize the uniform of a member of your crew. Could they be running after the intruder too? Is this just a big misunderstanding?`
```

				defer
			label follow
			`You can now clearly see it is indeed a member of your crew, not just a disguised mercenary or a stowaway, and you know them. The traitor is clearly heading to the exit, by another path than the one you came in by. You know your ship like your pocket, and you're aware it will take them a while. They haven't noticed you're following them yet.`
			choice
				`(Wait for the traitor near the exit to stun them.)`
					goto wait
				`(Order the crew member to stand down.)`
				`(Shoot.)`
					goto shoot
			`There is no doubt that you were heard, but the only reaction from the silouette is to run faster, and by another, more direct path. You cannot catch them, and you assume they escaped your ship, as later on you cannot find any intruders when doing a full search of your ship. Thankfully, they didn't manage to steal anything, surprisingly they only moved a few gold bars.`
				accept
			label shoot

> 			`You hit the crew member in the shoulder and they fall. You go remove their weapon to be safe and ask for an explanation, but the only answer you get is "nemo me impune lacessit" before they lose consciousness. Looking into their eyes, you could see terror. You put them at a nearby hospital, but when you come back later, the crew member is gone, and no one has seen or heard anything.`

Reviewer: ```suggestion
			`You hit the crew member in the shoulder and they fall. You remove their weapon to be safe and ask for an explanation, but the only answer you get is "nemo me impune lacessit" before they lose consciousness. Looking into their eyes, you could see terror. You put them at a nearby hospital, but when you come back later, the crew member is gone, and no one has seen or heard anything.`
```

			`In order to be sure to avoid any misunderstanding, you shout "Hello?" to see if someone answers. Nobody does. You still head into the cargo bay, a bit more alert than before. As you enter, you see that a few gold bars have fallen off the pile. That was probably the sound you heard.`
			choice
				`(Head in that direction.)`
				`(Call for help.)`
					goto help
			`You move that way, but soon you feel a pressure on the back of your neck. "Sweet dreams, Captain." You recognize the voice of one of your crew before you are incapacitated by a quick hit to your head. When you wake up, some of the bars are missing, and so are two members of your crew.`
				goto lostbars
			label wait
			`You patiently wait for the intruder to arrive and hit them in the back with your pistol. Given it is a member of your crew that betrayed you, you decide not to involve anyone else, as they could be complicit. You first remove the weapon they are carrying. After doing a thorough search, you notice a distinct marking on the crew member's leg, which is only visible by a special ultraviolet light. This seems bad, very bad. Then you pause for a second to wonder if you really want to know about this.`

> 			choice

Reviewer: Wait, what's the flow of this choice? Shouldn't you have a 'goto bliss' or something after the first option, instead of throwing water at them?

				`(Ignorance is bliss.)`
				`(I prefer not to stay in the dark. Wake them.)`
					goto light

> 			`You throw a bucket of water on the crew member, waking them up.`

Reviewer: ```suggestion
			`You splash water on the crew member, waking them up.`
```
Throwing a bucket at them might be excessive.

			label bliss
			`You leave the traitor here on this world, and they seem thankful but deeply troubled by what happened. You wonder what could have attracted such attention to you. Whatever it was should probably be safely stashed away, or gotten rid off.`
				accept
			label light
			`You throw a bucket of water on the crew member. They wake up, but at first refuse to tell you anything. Once you reveal you're aware of their marking, they concede to reveal something to you. They seem to recognize the hopelessness of their situation, and may be willing to bargain. "This sign means that I hold my life to someone, and have to do one task, any task, to repay them. There is something here, on this ship, of the highest importance to some... I may not, and will not say anything else, no matter what you try to make me do it. I am dead already, and they own me."`
			`You nod, and they then proceed, "I will tell you the truth, but then you will let me go and we will never see each other again. But let me be clear, I will not speak, under any circumstances other than those."`
			choice
				`(This is it. I want to know all.)`
				`(Ignorance is bliss.)`
					goto bliss

> 			`They takes a deep breath. "An important organization wants these gold bars you have on board. They're not just normal gold. If you were to sell them to me, I could give you a very generous price of 1000000 credits per bar, and you wouldn't be bothered by this again. 1 million per bar. You see, something is within these bars, something of the highest importance, and they want all of them. They will not stop until they have every single one. I may have told you too much for them to allow me to live already."`

Reviewer: ```suggestion
			`They take a deep breath. "An important organization wants these gold bars you have on board. They're not just normal gold. If you were to sell them to me, I could give you a very generous price of 1000000 credits per bar, and you wouldn't be bothered by this again. 1 million per bar. You see, something is within these bars, something of the highest importance, and they want all of them. They will not stop until they have every single one. I may have told you too much for them to allow me to live already."`
```

		payment 1000000
	npc
		# the Alphas and their minions let the Mercenaries alone so that this would happen. They probably even gave them a tip about it, but not in person.
		government "Mercenaries"
		system "Waypoint"
		personality disables plunders staying harvests
		fleet "Large Northern Pirates" 5
	npc
		government "Pirate"
		system "Waypoint"
		personality entering uninterested plunders disables coward
		fleet "Small Northern Pirates" 5
	to complete
		has "Unfettered: Escort Hai Fleet: failed"
		not "Unfettered: Escort Hai Fleet: aborted"
	on abort
		"reputation: Mercenaries" <?= -1000
	on complete
		"reputation: Mercenaries" <?= -1000
		clear "unfettered: stopped on Greenbloom"
		log `The plan was to escort a Hai fleet carying supplies and passengers into Unfettered space but an opportunity to feed them to the wolves instead was presented, and taken.`
		# The shipment gets lost faster, because they're busy shipping all of these supplies to Unfettered space and can't dedicate as much resources to stealing Alpha supplies.
		event "unfettered: shipment lost" 30
		set "unfettered: hunger"
		# Make the hunger last around 20 days longer.

> 		"unfettered: hunger stops" = -500

Reviewer: Is this 20 days _longer_?

Author: yeah on each day it is increased by random; 0-99 which on average is 50

also the Solifuge and the Hai would have done a better job at it than mercenaries, this incident made it so less Hai want to help the Unfettered, separating the two further (which the mercenaries want)

Reviewer: Yeah, but shouldn't it be += if you are making it longer?

Author: it could already be in progress so I want to reset it if thats the case

		payment 4000000
		dialog `As before, a voice addresses you once you are alone in your quarters. "We have transferred 4 million credits to your account, and made sure the elder will not bother you. Do be aware, however, that a food situation has arrisen, not because of our small operation, but because of the bomb's presence in the first place. The Unfettered will probably be busy for some time... About the governor, if it can ease your conscience, we only captured her."`

# secret mission
# happens if you accept a duel in "Unfettered: Materials Intercepted" (offered if your fleet is smaller than 2 heavy warships)
mission "Mercenaries: Duel Squared"
	landing
	source "New Tortuga"
	name "Duel with the twins."
	description "Beat the two twins today, or lose 10 million credits!"
	to offer
		has "mercenaries: accepted duel"
		has "Unfettered: Cavalry Arrives: done"
	on offer

> 		dialog `The two mysterious ships hail you, "We are ready for a fight in orbit whenever you are. If you want, you can still call this off... At a cost."`

Reviewer: ```suggestion
		dialog `The two mysterious ships hail you, "We are ready for a fight in orbit whenever you are. If you want, you can still call this off... at a cost."`
```

	# super high bribe for when they attack you
	government "Mercenaries"
		bribe 2
	# can't rep farm anymore
	government "Shadow"
		"attitude toward"
			"Cerberus" -.01

event "mercenaries: joined"
	government "Escort"
		swizzle 9

# they get mad when you kill the twins, striking at you after they had time to muster their strength.
# the Hai did not dare to contest them
mission "Mercenaries: Daemons Unleashed"
	landing
	invisible
	to offer
		has "event: mercenaries: thorn talon vengeance"
	npc kill
		system "Wah Yoot"
		government "Shadow"
		personality heroic staying nemesis vindictive target mute
		ship "Antlion" "Parmen"
		conversation
			`As soon as you destroy the flagship of this fleet, the others cease firing. The new flagship hails you. "I am now in command of the Mercenaries. I will stop making the same mistake my predecessor always did, right now, before this becomes an uncontrollable spiral of destruction. I have decided to make you a part of our organization. Keep the bars you have safe, <last>, that is all I ask of you."`
			choice
				`	"May I wear your colors as well?"`
					goto wearcolors
				`	"How can I restore my reputation with the Unfettered?"`

> 				`	End the connection.`

Reviewer: ```suggestion
				`	(End the connection.)`
```

		to despawn
			has "mercenaries: took out leader"
		fleet "Marauders" 8
	on enter "Wah Yoot"
		dialog `As you enter Wah Yoot you spot a fleet of mercenaries, larger than anything you've seen from them before. The captain of the fleet hails you. "Nemo Me Impune Lacessit, <first> <last>. You have killed our own. We have come to bring justice upon you, whatever the cost. There will be no negotiations, no forgiveness. I am the Shadow of Mintaka, and the last person you will ever meet."`

ship "Corvette" "Corvette (Harpy)"
	outfits
		"Hai Tracker Pod" 4
		"Hai Tracker" 112
		"Heavy Anti-Missile Turret" 2
		"Fission Reactor"
		"LP036a Battery Pack"
		"D94-YV Shield Generator"
		"Cooling Ducts"
		"Bunk Room"
		"Pulse Rifle" 3
		"Brig" 2
		`"Biroo" Atomic Thruster`
		`"Bondir" Atomic Steering`
		"Hyperdrive"
	cargo
		outfits
			"Gold Bars" 15

# happens when you land on the world instead of taking the cube in orbit, either bargain for it or steal it (or the wrong item)
mission "Unfettered: Special Cargo 1B"
	landing
	invisible
	source "Stormhold"
	destination "Warfeed"
	to offer
		has "Unfettered: Special Cargo: active"
		not "outfit (all): Mystery Cube"
	on offer
		conversation

> 			`You land on the planet next to another empty spot, hoping the Degenerate will land there. She does not seem to want to take any chances, and lands near a few of her own warships. As you depart the ship, your hairs stand up on the back of your neck. You try to get close to the Degenerate without drawing too much attention to yourself, and eventually arrive.`

Reviewer: ```suggestion
			`You land on the planet next to another empty spot, hoping the Degenerate will land there. She does not seem to want to take any chances, and lands near a few of her own warships. As you depart the ship, your hairs stand up on the back of your neck. You try to get close to the Degenerate without drawing too much attention to yourself, and eventually arrive there, undetected.`
```

			choice
				`	(Try to steal the item.)`
					goto steal
				`	(Try to bargain.)`
					goto bargain
				`	(Try to take an isolated pirate captive and ask for the item in exchange.)`

> 			`	In front of the ship there's a group of pirates, and you see one of them separate from the others to go within the ship. Judging from his appearance and the way others where addressing him, you assume him to be the leader of that small band, at least. You sneak past the group in front and follow him.`

Reviewer: ```suggestion
			`	In front of the ship there's a group of pirates, and you see one of them separate from the others to go within the ship. Judging from his appearance and the way others were addressing him, you assume him to be the leader of that small band, at least. You sneak past the group in front and follow him.`
```

			choice
				`	(Search the lower decks.)`
				`	(Search close to the weapons stations.)`
					goto weapons
				`	(Search the bridge.)`
					goto bridge
				`	(Look near the engines.)`
					goto engine
				`	(Check among the cargo.)`
					goto cargo
				`	(Search the upper decks.)`
				`	(Leave the ship before you get caught and provoke the warlord.)`
					goto challenge
			`	You notice a lot of security stations around, they obviously care a lot about their warlord's security.`
				goto caught
			label weapons
			`	This part of the ship is mostly full of torpedoes - apparently the warlord loves them. But there are not many people here, since it does not seem to have suffered damage. The torpedoes will probably be restocked soon, so you leave before someone comes in and notices you.`
				goto found
			label bridge
			`	This bridge is very luxurious, you see that this warlord likes comfort, for herself at least. But you cannot find anything that looks unique at all.`
				goto caught
			label cargo

> 			`	There are a lot of exotic things here, most of which are probably not legal at all. You better choose what to take with you, and fast! You also notice the presence of gold bars. There's no way you could carry all of them back with you discreetly, but you can drop them off the ship and then go grab them once you are out, hoping nobody will notice. Looking through a porthole you don't see anyone around at the moment. Conveniently, there is even some sand on the ground there, so the bars should sink in, and be harder to notice.`

Reviewer: ```suggestion
			`	There are a lot of exotic things here, most of which are probably not legal at all. You better choose what to take with you, and fast! You also notice the presence of gold bars. There's no way you could carry all of them back with you discreetly, but you can drop them off the ship and then go grab them once you are out, hoping nobody will notice. Looking through a porthole, you don't see anyone around at the moment. Conveniently, there is even some sand on the ground there, so the bars should sink in, and be harder to notice.`
```

					goto takebars
			label wrongitem
			`	You take the item, hoping it is the right one, and swiftly leave the ship. If only you had been given a more precise description, you could know for sure. Some pirates seem to notice you, and maybe they suspect something or recognize this item, but nobody tries to stop you. As you reach your ship, you notice that the Degenerate just launched, right before you. They must have noticed the item was missing.`
				flee
			label takebars
			`	How many stacks of bars will you take? The more, the bigger the risk and reward...`
			choice
				`	(One will do.)`
					goto one
				`	(Three should be enough.)`
					goto three
				`	(Five.)`
					goto five
				`	(Eight could be reasonable.)`
				`	(Take as much as I can: fifteen!.)`
				`	(Never mind, this isn't a good idea.)`
					goto fake
			`	You got too greedy. As you're dropping the bars, you feel something cold hit your head. You never wake up again.`
				die
			label one
			action
				outfit "Gold Bars"
			``
				goto dropbars
			label three
			action
				outfit "Gold Bars" 3
			``
				goto dropbars
			label five
			action
				outfit "Gold Bars" 5
			label dropbars

> 			`	You drop the gold and then leave the ship. Afterwards, you go around the ship to the location where they are partially buried in the sand, and take them discreetly with some members of your crew. You have to silence one or two curious pirates, but that's about it, and you can now head back to your ship. As expected, they have noticed the theft. you see them launching before you can do so yourself. This will be a bumpy ride.`

Reviewer: ```suggestion
			`	You drop the gold and then leave the ship. Afterwards, you go around the ship to the location where they are partially buried in the sand, and take them discreetly with some members of your crew. You have to silence one or two curious pirates, but that's about it, and you can now head back to your ship. As expected, they have noticed the theft. You see them launching before you can do so yourself. This will be a bumpy ride.`
```

				flee
			label caught
			`	You do not manage to find the item here. However, you've attracted way too much suspicion by looking and opening crates everywhere you could. One local worker asks you something about mechanics. Being no engineer yourself, you cannot make any sense of what he says, even less formulate a correct response, and he calls a guard over you. You are captured and sold as a slave, condemned to a few years of hard work before a sad death.`

> 				die

Reviewer: ~~Time flies when you are having fun.~~

Author: ik right them years passed like its nothing

					goto investigate
			action
				outfit "Gold Bars" -5

> 			`	You let the disturbance happen, and after some more noise including a big metallic object falling, it stops. It could be worth it to check your cargo bay to see if everything is ok.`

Author: ~~This is how I get rid of ``~~
Yeah here is second attempt
```suggestion
			`	You decide against going to check it out now. A few moments later, you hear a big metallic sound, before the calm returns. You may want to check your cargo bay later on.`
```

> 			`	You let the disturbance happen, and after some more noise including a big metallic object falling, it stops. It could be worth it to check your cargo bay to see if everything is ok.`

Reviewer: Added a third attempt.

> 			`	You decide against going to check it out now. A few moments later, you hear a big metallic sound, before the calm returns. You may want to check your cargo bay later on.`

Reviewer: ```suggestion
			`	You decide not to check it out now. A few moments later, you hear a big metallic sound, followed by deep silence. You may want to check your cargo bay later on.`
```

			action
				set "mercenaries: brokered deal"
				"reputation: Mhairi's Legion" >?= 1

> 			`	You go to your ship as planned, preparing to the planet as planned, and no pirates try to stop you on the way. It seems like they were mens of their words.`

Author: Too many as planned lol 
```suggestion
			`	You head to your ship, preparing to leave the planet. No pirates try to stop you on the way. In fact, the pirates in orbit are no longer hostile to you. It seems like they respected the agreement.`
```
Like this?

> 			`	You go to your ship as planned, preparing to the planet as planned, and no pirates try to stop you on the way. It seems like they were mens of their words.`

Reviewer: Yes, that's good.
